docs/releases/1.11.txt

```diff
@@ -399,6 +399,10 @@ Miscellaneous
 * The :djadmin:`makemessages` command now requires configured settings, like
   most other commands.
 
+* ``utils.http.is_safe_url()`` accepts a new ``secure`` argument. If
```

Comment:
I'm not sure this needs to be mentioned in backwards-incompatible changes changes? Maybe Django's usage of the new argument could be considered backwards-incompatible but that's not mentioned. By the way, I'd make Django's usage of the new argument in a separate commit. You can use "Refs #26902 --" for that commit message.